#0 - Basic Settings

In [ ]:
#Permission to access any Google Drive file
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
#increasing the ability to view columns and rows
import pandas as pd

pd.set_option('display.max_columns', 7000)
pd.set_option('display.max_rows',70000)

##1 - Reading *wild* protein files

You must select **Query 1** or **Query 2**.
In my project, **Query 2** was used

##1.1 Query 1

**Wild-type** proteins were obtained from the PDB website (https://www.rcsb.org/). Select **Advanced Search**.

In advanced search, create the following query:

Experimental Method = **X-RAY DIFFRACTION** **AND**

Data Collection Resolution **< 2.6**   **AND**

Source Organism Taxonomy ID = **9606** **AND**

Polymer Entity Mutation Count = **0**  **AND**

Coverage **>= 0.05**

With this we are selecting all wild PDBs obtained, through X-Ray, and with a precision (in Angston) less than 2.6 and that are human.

The query generated 6077 PDBs.

Download made on 03/25/2021

The description of the attributes that can be used to assemble queries is available at: https://www.rcsb.org/docs/search/advanced-search/attribute-details

In [ ]:
import pandas as pd
df = pd.read_csv('drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/PDB_wild_Query.txt', header=None, sep=',')

In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1 entries, 0 to 0
Columns: 6077 entries, 0 to 6076
dtypes: float64(13), object(6064)
memory usage: 47.6+ KB


In [ ]:
df.head()

0     1     2     3     4     5     6     7     8     9     10    11    \
0  3BLJ  3A4S  3A4U  3A99  3A8Y  3AB3  3ABD  3ADL  3AGV  3AGM  3AJB  3AL3   

   12    13    14    15    16    17    18    19    20    21    22    23    \
0  3AL5  3B08  3AP2  3AP1  3B0A  2ZWL  2ZZU  3AQA  3AQE  3B4D  3B3Q  3B3I   

   24    25    26    27    28    29    30    31    32    33    34    35    \
0  3AU4  3AVA  3AVB  3AVC  3AVF  3AVG  3AV9  3B6S  3B76  3AVH  3AVI  3AVJ   

   36    37    38    39    40    41    42    43    44    45    46    47    \
0  3AVK  3AVL  3AVM  3AVN  3B7O  3B9C  3B9F  3BC1  3BCH  3BC5  3BER  3BGQ   

   48    49    50    51    52    53    54    55    56    57    58    59    \
0  3BH8  3BGM  3BHB  3BHD  3BH9  3BGZ  3BIU  3BIV  3BK0  3QO2  3QR2  3R27   

   60    61    62    63    64    65    66    67    68    69    70    71    \
0  3QTO  3R66  3R5J  3Q8P  3Q8Q  3Q92  3QA2  3QAH  3QBY  3QC3  3QGY  3QGW   

   72    73    74    75    76    77    78    79    80    81    82    83    \
0  3R6G  3R6L  3R6B  3QTV  3R7S  3R7N  3R7B  3R8Q  3QX3  3R93  3QX5  3QWC   

   84    85    86    87    88    89    90    91    92    93    94    95    \
0  3QZT  3QZV  3QZS  3RAW  3RC4  3RC5  3RBN  3RBQ  3RDD  3RCO  3RD2  3REE   

   96    97    98    99    100   101   102   103   104   105   106   107   \
0  3RFE  3REZ  3RF3  3RH4  3RH5  3RH6  3RGH  3RIG  3RJE  3RJF  3RJG  3RJH   

   108   109   110   111   112   113   114   115   116   117   118   119   \
0  3RJI  3RJJ  3RIY  3RK6  3RJK  3RJM  3RK2  3RL7  3RL8  3RML  3RMM  3RLW   

   120   121   122   123   124   125   126   127   128   129   130   131   \
0  3RLY  3RM0  3RM2  3RNJ  3RNM  3RNN  3RMN  3RMO  3RNY  3ROV  3RQD  3RQ7   

   132   133   134   135   136   137   138   139   140   141   142   143   \
0  3ODI  3ODL  3OES  3OG4  3OG6  3NOI  3O0I  3NO8  3O2Q  3NR3  3O2S  3O2T   

   144   145   146   147   148   149   150   151   152   153   154   155   \
0  3O35  3O36  3NRU  3O3A  3O3B  3O3D  3O3E  3O70  3O5Z  3O7V  3O7A  3NWN   

   156   157   158   159   160   161   162   163   164   165   166   167   \
0  3O8I  3NY1  3NY3  3NY5  3NZL  3OA6  3OB9  3OB1  3OB2  3OBX  3OD8  3OD5   

   168   169   170   171   172   173   174   175   176   177   178   179   \
0  3OJ3  3OLH  3OMC  3OMG  3OLR  3P0N  3OP0  3P0P  3P0Q  3P0C  3P1M  3P1N   

   180   181   182   183   184   185   186   187   188   189   190   191   \
0  3P1O  3P1P  3P1Q  3P1R  3P1S  3P1I  3P1A  3P17  3OQS  3P34  3P35  3OQ5   

   192   193   194   195   196   197   198   199   200   201   202   203   \
0  3OS5  3P45  3P36  3P37  3P2Z  3P4U  3OSN  3OSE  3P4F  3P70  3OV1  3P6C   

   204   205   206   207   208   209   210   211   212   213   214   215   \
0  3P6D  3P6E  3P6F  3P6G  3P6H  3OW8  3OVE  3P7F  3P7G  3P7H  2VDR  2VGQ   

   216   217   218   219   220   221   222   223   224   225   226   227   \
0  2VIF  2VX2  2VX3  2W7R  2VX8  2VXW  2WEV  2WFY  2VNF  2VPE  2VPG  2W2I   

   228   229   230   231   232   233   234   235   236   237   238   239   \
0  2VRF  2W2M  2VSL  2W3O  2VTB  2W4O  2W5Z  2WFJ  2WH6  2WH0  2WNA  2WO2   

   240   241   242   243   244   245   246   247   248   249   250   251   \
0  2WO3  2X04  2WO1  2WPH  2X1H  2WPI  2WPJ  2X2C  2WPK  2WPL  2X4N  2X4P   

   252   253   254   255   256   257   258   259   260   261   262   263   \
0  2X4Q  2X4T  2WUH  2X70  2X7K  2X7H  2WY7  2WY8  2XAB  2XDX  2XDS  2XDU   

   264   265   266   267   268   269   270   271   272   273   274   275   \
0  2XDK  2XDL  2XHR  2XHT  2XHI  2XJX  2XK2  2XJG  2XJJ  2Y0B  2XPX  2Y1N   

   276   277   278   279   280   281   282   283   284   285   286   287   \
0  2XRI  2XRA  2XR5  2XQQ  2XS6  2Y4V  2XSQ  2XSS  2Y7J  2XV5  2Y8L  2YLE   

   288   289   290   291   292   293   294   295   296   297   298   299   \
0  2YLF  2YMT  2Z0B  2Z14  2Z15  2Z17  2Z0W  2YQ6  2YQ7  2Z5D  2Y9Q  2XX3   

   300   301   302   303   304   305   306   307   308   309   310   311   \
0  2YVI  2YV8  2Z6W  2YWK  2YXJ  2YXM  2YY1  2Z9A  

In [ ]:
base_PDB_wild = df.transpose()

In [ ]:
base_PDB_wild.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 6077 entries, 0 to 6076
Data columns (total 1 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   0       6077 non-null   object
dtypes: object(1)
memory usage: 255.0+ KB


In [ ]:
base_PDB_wild.head()

0
0  3BLJ
1  3A4S
2  3A4U
3  3A99
4  3A8Y

In [ ]:
#Rename column 0 to PDB_wild_id
base_PDB_wild.rename(columns={0: 'PDB_wild_id'}, inplace=True)

In [ ]:
base_PDB_wild.head(20)

PDB_wild_id
0         3BLJ
1         3A4S
2         3A4U
3         3A99
4         3A8Y
5         3AB3
6         3ABD
7         3ADL
8         3AGV
9         3AGM
10        3AJB
11        3AL3
12        3AL5
13        3B08
14        3AP2
15        3AP1
16        3B0A
17        2ZWL
18        2ZZU
19        3AQA

In [ ]:
#Identify duplicates records in the data
dupes=base_PDB_wild.duplicated()
sum(dupes)

0

##1.2 Query 2

**Wild-type** proteins were obtained from the PDB website (https://www.rcsb.org/). Select **Advanced Search**.

In advanced search, create the following query:

Experimental Method = **X-RAY DIFFRACTION** **AND**

Refinement Resolution = **(range upper inclusive) 0 to 2.6** **AND**

Data Collection Resolution **(range upper inclusive) 0 to 2.6** **AND**

Source Organism Taxonomy ID = **9606** **AND**

Polymer Entity Mutation Count = **0**  

With this we are selecting all the wild PDBs obtained, through X-Ray, and with a precision (in Angston) less than 2.6 and that are human.
The query generated 28802 PDBs, distributed across two files.

Download made on 04/15/2021

The description of the attributes that can be used to assemble queries is available at: https://www.rcsb.org/docs/search/advanced-search/attribute-details

In [ ]:
import pandas as pd
df1 = pd.read_csv('drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/pdb_ids_00001-25000.txt', header=None, sep=',')
df2 = pd.read_csv('drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/pdb_ids_25001-28802.txt', header=None, sep=',')

In [ ]:
df1.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1 entries, 0 to 0
Columns: 25000 entries, 0 to 24999
dtypes: float64(78), object(24922)
memory usage: 195.4+ KB


In [ ]:
df2.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1 entries, 0 to 0
Columns: 3801 entries, 0 to 3800
dtypes: float64(20), object(3781)
memory usage: 29.8+ KB


In [ ]:
df1.head()

0     1     2     3     4     5     6     7     8     9     10    11     \
0  2V0A  2V01  2V02  2V0V  2V17  2V0Z  2V0O  2UP1  2V0R  2V0S  2V14  2V1W   

  12    13    14    15    16    17    18    19    20    21    22    23     \
0  2V1X  2V1Y  2V22  2V24  2V3H  2V3D  2V3E  2V3F  2V2W  2V2X  2V30  2V4L   

  24    25    26    27    28    29    30    31    32    33    34    35     \
0  2V4M  2USN  2V4B  2V3O  2V40  2V3Q  2V3S  2V60  2V61  2V5E  2V5F  2V4U   

  36    37    38    39    40    41    42    43    44    45    46    47     \
0  2UUJ  2UUK  2UUE  2UUF  2UUH  2V6H  2UUI  2V5Z  2V5Q  2V62  2V5T  2V66   

  48    49    50    51    52    53    54    55    56    57    58    59     \
0  2V5W  2QJ9  2QIU  2QKB  2QKI  2QKH  2QJR  2QK9  2QK5  2QLB  2QLL  2QLJ   

  60    61    62    63    64    65    66    67    68    69    70    71     \
0  2QKQ  2QL9  2QL7  2QL5  2QME  2QMD  2QMK  2QMJ  2QMG  2QMF  2QLR  2QLY   

  72    73    74    75    76    77    78    79    80    81    82    83     \
0  2QLU  2QND  2QNK  2QMS  2QMZ  2QMY  2QOC  2QOB  2R0B  2QOA  2R0N  2R0L   

  84    85    86    87    88    89    90    91    92    93    94    95     \
0  2QO7  2R05  2R03  2QO2  2R02  2QO9  2QO8  2QPJ  2QP8  2QP6  2R0U  2R15   

  96    97    98    99    100   101   102   103   104   105   106   107    \
0  2R13  2QOQ  2QQI  2QR2  2R2Q  2R32  2R2P  2QQO  2R2O  2QQN  2R2N  2QQM   

  108   109   110   111   112   113   114   115   116   117   118   119    \
0  2R2M  2QQJ  2R2J  2QPW  2QQ8  2R28  2R1U  2QQ5  2R24  2R1Q  2R1Z  2R3J   

  120   121   122   123   124   125   126   127   128   129   130   131    \
0  2R3I  2R3H  2R3G  2R3F  2R3R  2R3Q  2R3P  2R3O  2R3N  2R3M  2R3L  2R3K   

  132   133   134   135   136   137   138   139   140   141   142   143    \
0  2QR9  2R36  2R35  2R34  2QQR  2QRA  2R3A  2QSE  2R53  2QSQ  2R52  2QT1   

  144   145   146   147   148   149   150   151   152   153   154   155    \
0  2QT0  2QS9  2R3V  2R4B  2R5G  2R64  2QU3  2QU2  2QSZ  2QSY  2R59  2QT7   

  156   157   158   159   160   161   162   163   164   165   166   167    \
0  2R4V  2R55  2QUJ  2QUI  2QUH  2QUG  2QV4  2QV2  2R6N  2QTZ  2QTW  2QTU   

  168   169   170   171   172   173   174   175   176   177   178   179    \
0  2R7G  2QX8  2QX6  2R8U  2QX4  2R9P  2QY0  2R9O  2R9N  2QXM  2R9M  2QXJ   

  180   181   182   183   184   185   186   187   188   189   190   191    \
0  2QXI  2QXW  2QY7  2QWX  2QX9  2R99  2QXH  2QXG  2R9A  2QYQ  2QYP  2QYN   

  192   193   194   195   196   197   198   199   200   201   202   203    \
0  2QYM  2QYL  2QYK  2QZ4  2QYF  2QZO  2RA4  2RA3  2RA0  2RAW  2RB3  2RAH   

  204   205   206   207   208   209   210   211   212   213   214   215    \
0  2RAP  2RAJ  2RNF  2ROY  2ROX  2SEB  2SSP  2TCL  2RCZ  2RD7  2RD6  2RCT   

  216   217   218   219   220   221   222   223   224   225   226   227    \
0  2RCS  2TGI  2RCQ  2RE9  2RDW  2RDU  2RDT  2RED  2REA  2THF  2REY  2REX   

  228   229   230   231   232   233   234   235   236   237   238   239    \
0  2REW  2REI  2RF0  2REN  2RFX  2RG8  2RGE  2RFI  2RG6  2RG5  2RFS  2RFN   

  240   241   242   243   244   245   246   247   248   249   250   251    \
0  2RHE  2RGU  2RGP  2RHY  2RHX  2RIE  2RID  2RIC  2RIB  2RIA  2RHK  2RHI   

  252   253   254   255   256   257   258   259   260   261   262   263    \
0  2RI7  2RHU  2RIZ  2RIY  2RJF  2RJE  2RJD  2RJC  2RJ1  2RJ0  2RII  2RIX   

  264   265   266   267   268   269   270   271   272   273   274   275    \
0  2RJ9  2RIW  2RJ8  2RIV  2RJ7  2RJ6  2RIT  2RJ5  2RJ4  2RIQ  2RJQ  2RL0   

  276   277   278   279   280   281   282   283   284   285   286   287    \
0  2RKZ  2RKY  2RMB  2RMA  2QJF  2QJ2  6GXE  6GWN  6GWQ  6GWP  6H8P  6GWR   

  288   289   290   291   292   293   294   295   296   297   298   299    \
0  6H8R  6GX6  6GWJ  6GY5  6GY4  6H9J  6GXK  6GZD  6GZ6  6GYT  6GZY  6GZS   

  300   301   302   303   304   305   306   307   308   309   310   311    \
0  6GZJ  6GZL  6HAL  6HA6  6HBN  6HAX  6HB9  6HAZ 

In [ ]:
df2.head()

0     1     2     3     4     5     6     7     8     9     10    11    \
0  4RXA  4RZ9  4RYL  4RZ1  4RYC  4RYD  4RXZ  4RZA  4THN  4TKF  4TKG  4TKH   

   12    13    14    15    16    17    18    19    20    21    22    23    \
0  4TKI  4TKJ  4TJZ  4TKB  4TK5  4TJU  4TJW  4TJY  4TK0  6GM9  6GLW  6GLE   

   24    25    26    27    28    29    30    31    32    33    34    35    \
0  6GLB  6GLA  6GN5  6GN8  6GMX  6GMN  6GN1  6GMR  6GME  6GMC  6GMB  6GNY   

   36    37    38    39    40    41    42    43    44    45    46    47    \
0  6GNQ  6GNK  6H0W  6H0Y  6H0X  6H0Z  6H10  6H11  6H0U  6GOT  6H15  6H0A   

   48    49    50    51    52    53    54    55    56    57    58    59    \
0  6H0B  6H0E  6GOD  6H29  6GPW  6GPY  6GPR  6GQ3  6H22  6GPT  6GQ6  6H1H   

   60    61    62    63    64    65    66    67    68    69    70    71    \
0  6GPJ  6H1I  6GPL  6H1D  6H1F  6H1E  6GR9  6H2Z  6GQQ  6GR2  6GQP  6H34   

   72    73    74    75    76    77    78    79    80    81    82    83    \
0  6GQR  6H33  6H2U  6GR8  6GR7  6H2V  6GQO  6GQD  6GRZ  6H3Q  6H42  6H45   

   84    85    86    87    88    89    90    91    92    93    94    95    \
0  6H47  6GRU  6H46  6H3M  6GRP  6GRO  6GRB  6GRA  6H4Z  6GTB  6GSS  6H4S   

   96    97    98    99    100   101   102   103   104   105   106   107   \
0  6H4R  6GT6  6H4U  6GT5  6H4T  6H4V  6H4Y  6H4X  6H4K  6H4L  6H4O  6H50   

   108   109   110   111   112   113   114   115   116   117   118   119   \
0  6H4Q  6H52  6H4P  6H51  6H6A  6GUC  6GUB  6GTT  6H5T  6GTU  6GU6  6H5X   

   120   121   122   123   124   125   126   127   128   129   130   131   \
0  6H5R  6GU2  6GVA  6H77  6GV0  6H6S  6H6R  6GUE  6GUH  6GUK  6GWE  6GBI   

   132   133   134   135   136   137   138   139   140   141   142   143   \
0  6GBH  6GD2  6GD1  6GC2  6GC5  6GBW  6GDM  6GE0  6GE3  6GDQ  6GDC  6GD3   

   144   145   146   147   148   149   150   151   152   153   154   155   \
0  6GF1  6GES  6GEK  6GEC  6GE5  6GE4  6GE6  6GDY  6GG4  6GFE  6GFI  6GFK   

   156   157   158   159   160   161   162   163   164   165   166   167   \
0  6GFB  6GFA  6GEV  6GGN  6GH1  6GH4  6GGG  6GGH  6GFX  6GG8  6GFZ  6GHP   

   168   169   170   171   172   173   174   175   176   177   178   179   \
0  6GHV  6GHG  6GHK  6GHN  6GHM  6GHA  6GHD  6GH8  6GH9  6GIU  6GJ0  6GID   

   180   181   182   183   184   185   186   187   188   189   190   191   \
0  6GJR  6GJQ  6GJS  6GJU  6GJW  6GJJ  6GJI  6GJL  6GJN  6GJM  6GJP  6GK0   

   192   193   194   195   196   197   198   199   200   201   202   203   \
0  6GJB  6GJD  6GJE  6GJH  6GKS  6GKU  6GKT  6GL9  6GKF  6GJY  6GVX  6H7Z   

   204   205   206   207   208   209   210   211   212   213   214   215   \
0  6H7Y  6GVF  6H7E  6GVJ  6GVL  6GVK  6GXB  6TF7  6TF6  6TEI  6TEL  6TEN   

   216   217   218   219   220   221   222   223   224   225   226   227   \
0  6TGG  6TFP  6THA  6TGK  6THZ  6TIA  6THW  6TI8  6THX  6TI9  6THP  6TJK   

   228   229   230   231   232   233   234   235   236   237   238   239   \
0  6TJJ  6TKH  6TKG  6TKI  6TKL  6TKK  6TKA  6TJT  6TJM  6TJN  6TJQ  6TLG   

   240   241   242   243   244   245   246   247   248   249   250   251   \
0  6TLM  6TLL  6TLA  6TKV  6TL7  6TKY  6TKN  6TKP  6TL1  6TL3  6TKQ  6TKS   

   252   253   254   255   256   257   258   259   260   261   262   263   \
0  6TLW  6TLV  6TLO  6TLQ  6TLP  6TLS  6TLR  6TLU  6TM6  6TLT  6TNO  6TNC   

   264   265   266   267   268   269   270   271   272   273   274   275   \
0  6TND  6TN9  6TN8  6TMP  6TN1  6TMO  6TN0  6TMQ  6TN2  6TN5  6TN4  6TN7   

   276   277   278   279   280   281   282   283   284   285   286   287   \
0  6TOJ  6TOI  6TOL  6TOK  6U0K  6TON  6TOM  6TOO  6U0A  6U0D  6TOF  6TOH   

   288   289   290   291   292   293   294   295   296   297   298   299   \
0  6TOG  6TNQ  6U04  6U07  6TQ0  6TQ2  6TQ1  6TPD  6TPF  6TOR  6U0Q  6TP5   

   300   301   302   303   304   305   306   307   308   309   310   311   \
0  6U15  6U17  6TOU  6U16  6TQP  6T5P  6T5Q  6T5V  

In [ ]:
base_PDB_wild1 = df1.transpose()

In [ ]:
base_PDB_wild2 = df2.transpose()

In [ ]:
base_PDB_wild1.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 25000 entries, 0 to 24999
Data columns (total 1 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   0       25000 non-null  object
dtypes: object(1)
memory usage: 1.0+ MB


In [ ]:
base_PDB_wild2.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 3801 entries, 0 to 3800
Data columns (total 1 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   0       3801 non-null   object
dtypes: object(1)
memory usage: 219.4+ KB


In [ ]:
base_PDB_wild1.head()

0
0  2V0A
1  2V01
2  2V02
3  2V0V
4  2V17

In [ ]:
base_PDB_wild2.head()

0
0  4RXA
1  4RZ9
2  4RYL
3  4RZ1
4  4RYC

In [ ]:
#Rename column 0 to PDB_wild_id1
base_PDB_wild1.rename(columns={0: 'PDB_wild_id'}, inplace=True)

In [ ]:
##Rename column 0 to PDB_wild_id2
base_PDB_wild2.rename(columns={0: 'PDB_wild_id'}, inplace=True)

In [ ]:
base_PDB_wild1.head(20)

PDB_wild_id
0         2V0A
1         2V01
2         2V02
3         2V0V
4         2V17
5         2V0Z
6         2V0O
7         2UP1
8         2V0R
9         2V0S
10        2V14
11        2V1W
12        2V1X
13        2V1Y
14        2V22
15        2V24
16        2V3H
17        2V3D
18        2V3E
19        2V3F

In [ ]:
base_PDB_wild2.head(20)

PDB_wild_id
0         4RXA
1         4RZ9
2         4RYL
3         4RZ1
4         4RYC
5         4RYD
6         4RXZ
7         4RZA
8         4THN
9         4TKF
10        4TKG
11        4TKH
12        4TKI
13        4TKJ
14        4TJZ
15        4TKB
16        4TK5
17        4TJU
18        4TJW
19        4TJY

In [ ]:
#Join the two bases
base_PDB_wild = base_PDB_wild1.append(base_PDB_wild2, ignore_index=True)

In [ ]:
base_PDB_wild.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 28801 entries, 0 to 28800
Data columns (total 1 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   PDB_wild_id  28801 non-null  object
dtypes: object(1)
memory usage: 225.1+ KB


In [ ]:
#Identify duplicates records in the data
dupes=base_PDB_wild.duplicated()
sum(dupes)

0

##1.3 Generating a file with the *PDB_wild* table

Generating a file from the execution of query 1 or 2.

In this project, we used the **Query 2** option, as this way we would have more PDB instances.

In [ ]:
base_PDB_wild.to_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/PDB_wild_sem_coverage.csv",sep='\t',index=False)


##2 - Reading the *Wild Proteins* Table generated with the old Advanced Search interface on the PDB website

The **wild proteins** were obtained from the PDB website (https://www.rcsb.org/). Select **Advanced Search**.

**Choose a Query Type**: **Wild Type Protein**.

**Percent coverage of UniProt sequence**: **>= 90%**

Set **Add Search Criteria**

**Choose a Query Type**: **Experimental Method** and set **X-Ray**

Set **Add Search Criteria**

**Choose a Query Type**: **Experimental Method** and set **X-Ray Resolution**, with values **0.0** e **2.6** type range [0.0, 2.6)

Set **check box** at the end of the page:  **Retrieve only representatives at 90%   sequence identity** and set **Submit Query** (In this project this option was not selected to guarantee a greater number of instances)

On the next page, select **ORGANISM**: **Homo sapiens** and set botão **Download Files** button

With this we are selecting all wild PDBs with coverage greater than 90%, obtained through X-Ray, and with a precision (in Angston) less than 2.5 and that is human.

In [ ]:
#transforming the list of wild PDBs (resulting from my Advanced Search) into a csv file

lista_pdb = ["1A12", "1A1X", "1ABW", "1AIN", "1AQK", "1AUK", "1AXN", "1AYE", "1BD8", "1BLX", "1BPX", "1BPY", "1BPZ", "1BY8", "1C4Z", "1CB5",
             "1CB6", "1CI4", "1CKS", "1D1S", "1D4A", "1D5I", "1DGF", "1DHS", "1E98", "1EFV", "1EFV", "1EK6", "1EP9", "1EVU", "1F05", "1F0Y",
             "1F5N", "1FJ2", "1FW1", "1G3M", "1GAF", "1GGZ", "1GNU", "1H2K", "1H7C", "1HDO", "1HDR", "1HK0", "1HU0", "1HVY", "1I0Z", "1I1N",
             "1I3D", "1IAT", "1IHB", "1IMJ", "1IRD", "1IRD", "1IVH", "1IVY", "1J1J", "1J55", "1J72", "1J99", "1JKG", "1JR2", "1JSG", "1JV1",
             "1K3Y", "1K8U", "1KAO", "1KFU", "1KMV", "1KWM", "1KXP", "1L9X", "1LJ5", "1LN1", "1LO6", "1LPB", "1LS6", "1LWV", "1LWW", "1LWY",
             "1MR8", "1N0X", "1N52", "1NE7", "1NNL", "1OLM", "1ONI", "1P49", "1P5F", "1P60", "1PIN", "1PJ3", "1PL8", "1PR9", "1PSR", "1QGR",
             "1QGV", "1QMV", "1QO5", "1QOI", "1QUQ", "1R3Q", "1R5L", "1RSZ", "1RX0", "1RZ4", "1S1P", "1S5O", "1SNZ", "1SO7", "1SQW", "1T3Y",
             "1TEV", "1TRN", "1TV9", "1TVA", "1TVG", "1U3W", "1U6T", "1UOH", "1UOU", "1UPK", "1UPT", "1US0", "1W3W", "1W6K", "1WAR", "1WLJ",
             "1WMA", "1WOU", "1WSR", "1WUU", "1WZ9", "1WZV", "1XCR", "1XD3", "1XG5", "1XW6", "1Y1A", "1Y8Q", "1YB5", "1YOV", "1YXM", "1YZG",
             "1Z68", "1Z83", "1ZD8", "1ZJ6", "1ZJM", "1ZMD", "1ZS9", "1ZSV", "2A14", "2A2C", "2A4D", "2A5D", "2AEB", "2AG5", "2AKZ", "2AMY",
             "2ANT", "2AOT", "2B3Y", "2B5G", "2B6H", "2BBW", "2BJN", "2BKA", "2BKQ", "2BLE", "2BO9", "2BOA", "2BR9", "2BZG", "2C3N", "2C6Q",
             "2C9V", "2CAR", "2CB8", "2CH5", "2CW6", "2D2Z", "2D8N", "2DC3", "2DFD", "2DFK", "2E9L", "2E9X", "2EGD", "2F8A", "2F9D", "2FDV",
             "2FMP", "2FMQ", "2FMS", "2FS6", "2FUE", "2FVL", "2FZE", "2GCG","2GDZ", "2GRN", "2GWH", "2H18", "2H6F", "2HGS", "2HHJ", "2HI4",
             "2HRB", "2HYI", "2HYI", "2HYV", "2HZ5", "2I99", "2I9G", "2ICJ", "2IFQ", "2IIK", "2ISO", "2ISP", "2IWZ", "2J0S", "2J0S", "2J3T",
             "2J3T", "2J6L", "2J8Z", "2JBH", "2JBM", "2JDF", "2NML", "2NNJ", "2NUT", "2NZ2", "2NZT", "2O02", "2O07", "2O23", "2O28", "2OB0",
             "2OBV", "2OJW", "2OK3", "2OK5", "2OPW", "2OYC", "2OZB", "2P66", "2PBD", "2PD6", "2PE4", "2PFR", "2PI2", "2PLA", "2PNR", "2PNY",
             "2PPN", "2PQT", "2PSN", "2PXI", "2Q4G", "2Q4Q", "2Q4V", "2QFA", "2QME", "2QNK", "2QS9", "2QT1", "2R2Q", "2R3V", "2R55", "2R5G",
             "2V3Q", "2V40", "2VD6", "2VGQ", "2VH7", "2VT0", "2VXO", "2W2J", "2WC8", "2WM3", "2WM8", "2WYA", "2WZB", "2X7H", "2X7K", "2XQQ",
             "2XSQ", "2XSX", "2Y1H", "2Y73", "2YB6", "2YXT", "2Z5K", "2Z5X", "2Z6O", "2ZB4", "3A1J", "3A1J", "3AAA", "3AKM", "3AL5", "3B1U",
             "3B6H", "3BBB", "3BCZ", "3BHD", "3BUV", "3BXN", "3C1V", "3C2K", "3C2L", "3C2M", "3C4M", "3C5J", "3C6K", "3CH4", "3CKL", "3COG",
             "3COS", "3CPT", "3CTZ", "3CZH", "3D4J", "3D9S", "3DAX", "3DDS", "3DLJ", "3DLX", "3DOE", "3DXB", "3E6I", "3E6U", "3E77", "3E9K",
             "3EDY", "3EI2", "3EI3", "3EML", "3ENP", "3EPY", "3EUF", "3EX7", "3EX7", "3EXE", "3EXE", "3F3S", "3F5O", "3F81", "3F9M", "3FD5",
             "3FED", "3FEY", "3FFM", "3FGQ", "3FM0", "3FM8", "3G4E", "3GDX", "3GJ0", "3GP0", "3GZC", "3H7H", "3H8K", "3HLT", "3HN3", "3HNC",
             "3HY3", "3I2B", "3I2N", "3IAR", "3II0", "3IIJ", "3IK7", "3IQU", "3ISB", "3ISC", "3ISD", "3ISQ", "3IWP", "3JPN", "3JPO", "3JPP",
             "3JPQ", "3JPR", "3JPS", "3JPT", "3JUC", "3K1Z", "3K9O", "3KAN", "3KJY", "3KSE", "3KVH", "3L50", "3L7U", "3LGD", "3LK9", "3LWK",
             "3MBY", "3MDG", "3MDI", "3MDZ", "3MMY", "3MSH", "3N80", "3N94", "3N9U", "3N9Y", "3NKS", "3NR1", "3NR3", "3NSO", "3NXA", "3O2S",
             "3O3U", "3O4R", "3OAI", "3OJM", "3ORH", "3OVP", "3OW8", "3P0F", "3P6D", "3P8C", "3P8C", "3PCV", "3PDN", "3PE6", "3PVN", "3Q27",
             "3QJ4", "3QWL", "3R66", "3R9M", "3RH4", "3RJE", "3RJF", "3RJG", "3RJH", "3RJI", "3RJJ", "3RJK", "3RPP", "3RZN", "3S3J", "3S4Y",
             "3S5J", "3S5O", "3S5Z", "3TBG", "3TFR", "3TFS", "3U0V", "3U9W", "3UBW", "3UZD", "3V5W", "3VG7", "3VHT", "3VOW", "3VTV", "3VTW",
             "3W0T", "3W4U", "3WAN", "3WBG", "3WJA", "3WSO", "3WVZ", "3ZNN", "3ZXF", "4A35", "4A60", "4A6D", "4AD9", "4AFQ", "4AIW", "4AL0",
             "4AOW", "4AQL", "4AU8", "4AVS", "4AWN", "4AY1", "4BC3", "4BVM", "4BX8", "4C0O", "4C2J", "4C5W", "4C9B", "4CCA", "4CDO", "4CQM",
             "4D4Z", "4DHX", "4DIR", "4DM9", "4DMB", "4DO9", "4DOA", "4DOB", "4DOC", "4DUR", "4DY0", "4E45", "4E5Y", "4EBB", "4EN3", "4ENZ",
             "4ERC", "4EWE", "4EXT", "4F0Z", "4F3T", "4F6U", "4FBJ", "4FC7", "4FDI", "4FGL", "4FJV", "4FTG", "4FWE", "4FYT", "4GEH", "4GPZ",
             "4GQB", "4GUK", "4GWG", "4H0W", "4H1S", "4HOQ", "4HOR", "4HOS", "4HOT", "4HRG", "4I5L", "4I5L", "4I8V", "4IGH", "4IMA", "4IN0",
             "4J5R", "4JGT", "4JOI", "4K84", "4KEG", "4KKI", "4KLD", "4KLE", "4KLF", "4KLG", "4KLH", "4KLI", "4KLJ", "4KLL", "4KLM", "4KLO",
             "4KLQ", "4KLS", "4KLT", "4KLU", "4KRE", "4KRF", "4KX7", "4KXT", "4KXV", "4L23", "4L2O", "4L57", "4L8U", "4LDQ", "4LEC", "4LG1",
             "4LLD", "4LLQ", "4LLY", "4LVF", "4LVS", "4M2Y", "4M3P", "4M47", "4M6R", "4MF2", "4MF8", "4MFA", "4MFC", "4MFF", "4MHQ", "4MHX",
             "4MIK", "4MJ2", "4MPG", "4MPM", "4MY2", "4N1M", "4N20", "4N6V", "4NDN", "4NE3", "4NLK", "4NLN", "4NM1", "4NM2", "4NXZ", "4NY8",
             "4O0C", "4O5C", "4O5E", "4O5K", "4O9M", "4OE5", "4OFC", "4ONM", "4ONM", "4P2H", "4PGQ", "4PGX", "4PGY", "4PH5", "4PHA", "4PHD",
             "4PHE", "4PHP", "4PL7", "4PX2", "4PYU", "4Q27", "4QGK", "4QTB", "4QYN", "4R26", "4RA2", "4RAO", "4RJF", "4RPX", "4RPY", "4RPZ",
             "4RQ0", "4RQ2", "4RQ3", "4RQ4", "4RQ5", "4RQ6", "4RQ7", "4RQ8", "4RQR", "4RT2", "4RT3", "4RWF", "4TKB", "4TLG", "4TQ1", "4TUP",
             "4TUQ", "4TUR", "4TUS", "4TWE", "4UAW", "4UAY", "4UAZ", "4UB1", "4UB2", "4UB3", "4UB4", "4UB5", "4UBB", "4UBC", "4UEL", "4UF4",
             "4UHW", "4UYB", "4V24", "4WLH", "4WMU", "4WMY", "4WUY", "4X4W", "4X60", "4X90", "4XMP", "4XNV", "4XNY", "4XO6", "4XWH", "4XZM",
             "4Y08", "4Y1U", "4Y2H", "4YDX", "4YE6", "4YMM", "4YMN", "4YMO", "4YN4", "4YNO", "4Z3K", "4ZK3", "5A1I", "5A7F", "5AM2", "5AN4",
             "5B6M", "5BMU", "5BOL", "5BOM", "5BPC", "5C2K", "5C40", "5C50", "5C9K", "5CCB", "5CCX", "5CR5", "5CVD", "5CX3", "5D94", "5DB7",
             "5DB8", "5DB9", "5DBA", "5DBB", "5DBC", "5DG2", "5DGO", "5DOT", "5E2M", "5EBB", "5EK3", "5EKD", "5EOM", "5EOZ", "5EPC", "5EWI",
             "5F19", "5F9S", "5FA2", "5GTU", "5GU6", "5H7N", "5H7Q", "5H9S", "5HBS", "5HHH", "5HHI", "5HK1", "5HKG", "5HWL", "5HYD", "5HZ5",
             "5I2C", "5IAA", "5IKO", "5IN4", "5INB", "5IRQ", "5J0O", "5J0P", "5J0Q", "5J0R", "5J0S", "5J0T", "5J0U", "5J0W", "5J0X", "5J0Y",
             "5J29", "5J2A", "5J2B", "5J2C", "5J2D", "5J2E", "5J2F", "5J2G", "5J2H", "5J2I", "5J2J", "5J2K", "5J41", "5JH5", "5K89", "5KQL",
             "5L0V", "5L8I", "5LE5", "5LE5", "5LE5", "5LE5", "5LE5", "5LE5", "5LE5", "5LE5", "5LE5", "5LG8", "5LOU", "5LPG", "5M4L", "5MJ3",
             "5ML3", "5MPO", "5MX1", "5N10", "5NAW", "5O5T", "5O6X", "5O9M", "5OGJ", "5OMP", "5QJC", "5SYN", "5SYT", "5T3P", "5T4E", "5TB8",
             "5TB9", "5TBA", "5TC6", "5TZV", "5U2R", "5U2S", "5U2T", "5U3A", "5U8G", "5U9H", "5UGN", "5UGO", "5UGP", "5UIW", "5UPP", "5URA",
             "5UZ0", "5V1F", "5V1G", "5V1H", "5V1I", "5V1N", "5V1O", "5V1P", "5V1R", "5V7I", "5V80", "5V86", "5VBL", "5VCC", "5VEZ", "5VI6",
             "5VR8", "5VRW", "5VRX", "5VRY", "5VRZ", "5VS0", "5VS1", "5VS2", "5VS3", "5VS4", "5VWO", "5W0C", "5W8J", "5WBM", "5WNX", "5WNY",
             "5WNZ", "5WO0", "5WP1", "5X2L", "5XAD", "5XG7", "5XOC", "5XRG", "5Y1U", "5Y2S", "5Y7D", "5YTO", "5YVN", "5YVT", "5YX1", "5ZC9",
             "5ZQY", "5ZRT", "6AHV", "6AKW", "6ALJ", "6ARH", "6AT8", "6B1K", "6B3I", "6B3V", "6B9X", "6B9X", "6B9X", "6BCB", "6BCR", "6BD4",
             "6BEL", "6BEM", "6BEV", "6BKX", "6BSX", "6C01", "6C45", "6C6K", "6CLY", "6CPQ", "6CR4", "6CR5", "6CR6", "6CR7", "6CR8", "6CR9",
             "6CRB", "6CRC", "6CRH", "6CU9", "6CUA", "6CUB", "6CZK", "6D0A", "6D1U", "6DCV", "6DIA", "6E08", "6E1Z", "6E3R", "6E3V", "6E3W",
             "6E5W", "6E8Y", "6EOP", "6F7H", "6FAK", "6FCH", "6FD8", "6FIE", "6FOH", "6FW0", "6FW2", "6FXH", "6FXY", "6G28", "6G2Q", "6GD1",
             "6GID", "6GIU", "6GMC", "6GU2", "6H0B", "6H1E", "6H1E", "6H45", "6HOI", "6HYN", "6HYO", "6I18", "6I58", "6IIU", "6J4P", "6JPT",
             "6M7X", "6MJM", "6MNE", "6MO6", "6MR7", "6MS4", "6N2R", "6N2S", "6N2T", "6NDJ", "6O1G", "6Q21", "6Q49", "6Q4R", "6QAP", "6QY9",
             "7FAB", "9ICW", "9ICX"]
wild = open('drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/pdb_wild_versao_old.txt', 'w')
columnTitleRow = "PDB_wild_id\n"
wild.write(columnTitleRow)
for pdb in lista_pdb:
	row = pdb + "\n"
	wild.write(row)
wild.close()

In [ ]:
#open the PDB_wild.csv file in a dataframe

import pandas as pd

pdb_wild_old = pd.read_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/pdb_wild_versao_old.txt", sep='\t')

In [ ]:
pdb_wild_old.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 979 entries, 0 to 978
Data columns (total 1 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   PDB_wild_id  979 non-null    object
dtypes: object(1)
memory usage: 7.8+ KB


In [ ]:
pdb_wild_old.head(60)

PDB_wild_id
0         1A12
1         1A1X
2         1ABW
3         1AIN
4         1AQK
5         1AUK
6         1AXN
7         1AYE
8         1BD8
9         1BLX
10        1BPX
11        1BPY
12        1BPZ
13        1BY8
14        1C4Z
15        1CB5
16        1CB6
17        1CI4
18        1CKS
19        1D1S
20        1D4A
21        1D5I
22        1DGF
23        1DHS
24        1E98
25        1EFV
26        1EFV
27        1EK6
28        1EP9
29        1EVU
30        1F05
31        1F0Y
32        1F5N
33        1FJ2
34        1FW1
35        1G3M
36        1GAF
37        1GGZ
38        1GNU
39        1H2K
40        1H7C
41        1HDO
42        1HDR
43        1HK0
44        1HU0
45        1HVY
46        1I0Z
47        1I1N
48        1I3D
49        1IAT
50        1IHB
51        1IMJ
52        1IRD
53        1IRD
54        1IVH
55        1IVY
56        1J1J
57        1J55
58        1J72
59        1J99

In [ ]:
#Identify duplicates records in the data
dupes=pdb_wild_old.duplicated()
sum(dupes)

22

In [ ]:
pdb_wild_old = pdb_wild_old.drop_duplicates(['PDB_wild_id'],keep='first')

In [ ]:
pdb_wild_old.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 957 entries, 0 to 978
Data columns (total 1 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   PDB_wild_id  957 non-null    object
dtypes: object(1)
memory usage: 15.0+ KB


##2.1 Generating a file with the *PDB_wild_old* table without duplicate records

In [ ]:
pdb_wild_old.to_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/PDB_wild_old.csv",sep='\t',index=False)


#3 -  Comparing the old base with the new base

It will be checked whether the old database that has 957 records is contained in the new database (**Query 2** option) that has 28801 records

In [ ]:
len(base_PDB_wild)

28801

In [ ]:
len(pdb_wild_old)

957

In [ ]:
len(pdb_wild_old.merge(base_PDB_wild))

915

##1.4 Comparing the old base with the new base (using Display Results with the "Polymer Entities" option)

###1.4.1 Reading the file with wild PDBs

In [ ]:
import pandas as pd
df = pd.read_csv('drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/PDB_wild_Polimero_Query.txt', header=None, sep=',')

In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1 entries, 0 to 0
Columns: 4356 entries, 0 to 4355
dtypes: object(4356)
memory usage: 34.2+ KB


In [ ]:
df.head()

0       1       2       3     ...    4352    4353    4354    4355
0  3GDQ_1  3EMW_1  4AUQ_3  5Z94_2  ...  3U9H_1  4MSM_2  3BRT_1  5PCH_1

[1 rows x 4356 columns]

In [ ]:
base_PDB_wild_polimero = df.transpose()

In [ ]:
base_PDB_wild_polimero.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 4356 entries, 0 to 4355
Data columns (total 1 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   0       4356 non-null   object
dtypes: object(1)
memory usage: 228.1+ KB


In [ ]:
#Identify duplicates records in the data
dupes=base_PDB_wild_polimero.duplicated()
sum(dupes)

0

In [ ]:
base_PDB_wild_polimero.head()

0
0  3GDQ_1
1  3EMW_1
2  4AUQ_3
3  5Z94_2
4  2YBX_1

In [ ]:
#Rename column 0 to PDB_wild_id
base_PDB_wild_polimero.rename(columns={0: 'PDB_wild_id'}, inplace=True)

In [ ]:
base_PDB_wild_polimero.head(20)

PDB_wild_id
0       3GDQ_1
1       3EMW_1
2       4AUQ_3
3       5Z94_2
4       2YBX_1
5       6G6L_1
6       3CEK_1
7       5PCM_1
8       3KUL_2
9       3KUL_1
10      6WE9_2
11      5RKY_1
12      6MP4_1
13      5Z93_2
14      3KUI_1
15      5RKX_1
16      5PCL_1
17      6SWG_1
18      6GYT_2
19      6GYT_1

In [ ]:
pdb_wild_old.head(20)

PDB_wild_id
0         1A12
1         1A1X
2         1ABW
3         1AIN
4         1AQK
5         1AUK
6         1AXN
7         1AYE
8         1BD8
9         1BLX
10        1BPX
11        1BPY
12        1BPZ
13        1BY8
14        1C4Z
15        1CB5
16        1CB6
17        1CI4
18        1CKS
19        1D1S

It will be checked whether the old database that has 979 records is contained in the new database that has 4356 records

In [ ]:
len(base_PDB_wild_polimero)

4356

In [ ]:
len(pdb_wild_old)

957

In [ ]:
len(pdb_wild_old.merge(base_PDB_wild_polimero))

0

##1.5 Reading and integration of PDB_Wild Bases

In this section, the wild PDB bases associated with each tissue already processed and integrated into a single base will be read.

In [ ]:
#increasing the ability to visualize columns and rows
import pandas as pd

pd.set_option('display.max_columns', 7000)
pd.set_option('display.max_rows',90000)

In [ ]:
import pandas as pd

PDB_wild_01 = pd.read_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/BLCA_PDB_wild_id.csv",delimiter='\t')
PDB_wild_02 = pd.read_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/LUAD_PDB_wild_id.csv",delimiter='\t')
PDB_wild_03 = pd.read_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/LUSC_PDB_wild_id.csv",delimiter='\t')




In [ ]:
base_PDB_wild = PDB_wild_01.append([PDB_wild_02, PDB_wild_03], ignore_index=True)

In [ ]:
base_PDB_wild.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1253 entries, 0 to 1252
Data columns (total 1 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   PDB_wild_id  1253 non-null   object
dtypes: object(1)
memory usage: 9.9+ KB


In [ ]:
base_PDB_wild.head()

PDB_wild_id
0        4F14
1        6R15
2        6Y4O
3        5MY9
4        4UU5

In [ ]:
#Identify duplicates records in the data
dupes=base_PDB_wild.duplicated()
sum(dupes)

638

In [ ]:
#removing duplicates
base_PDB_wild_ok = base_PDB_wild.drop_duplicates(keep='first')

In [ ]:
#Identify duplicates records in the data
dupes=base_PDB_wild_ok.duplicated()
sum(dupes)

0

In [ ]:
base_PDB_wild_ok.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 615 entries, 0 to 1249
Data columns (total 1 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   PDB_wild_id  615 non-null    object
dtypes: object(1)
memory usage: 9.6+ KB


In [ ]:
#Put the PDBs in numerical order
df_sort = base_PDB_wild_ok.sort_values(by=['PDB_wild_id'], ascending=True)

In [ ]:
df_sort

PDB_wild_id
358         1A6Y
98          1D01
1140        1D0A
1030        1G1S
374         1I27
237         1J2X
365         1JID
252         1JR2
553         1L9X
738         1LKK
832         1LQV
272         1MG4
30          1MIZ
416         1MX3
15          1N0W
127         1OHE
89          1R6J
764         1SEN
208         1SYX
42          1T15
226         1TW6
132         1UFI
111         1UPT
400         1W4R
745         1W70
116         1WMH
167         1XED
698         1XGW
209         1XMK
833         1YB5
355         1YRK
396         1YTV
273         1Z8G
737         1ZD9
292         1ZIV
217         1ZKC
251         1ZSX
1124        1ZUO
366         2A1I
410         2A5J
1139        2AF0
317         2AHE
566         2ARR
364         2AUG
1083        2AWF
90          2B0O
422         2B3H
160         2BBW
229         2C2N
250         2C43
399         2C46
94          2CIA
146         2CXK
31          2D1X
758         2DFD
534         2E9S
190         2EK1
436         2ESL
320         2F5J
812         2F9N
248         2FH5
183         2FK9
26          2FLU
677         2FYT
267         2FZP
649         2G30
1206        2GAO
235         2GF9
287         2GZV
36          2H4V
158         2HDZ
405         2HE2
91          2HE9
818         2HHL
177         2HQ6
200         2HT9
378         2HUP
175         2I3H
255         2I3Y
347         2I7A
1099        2IIK
360         2IL1
824         2JKU
323         2NQA
342         2O52
225         2O9S
1197        2OBH
587         2OVC
11          2OVR
816         2P31
109         2P3W
75          2P57
1161        2P5S
274         2PFI
343         2PIE
135         2PNY
434         2Q3G
920         2QFE
395         2QFZ
786         2QIC
732         2QKH
425         2QT1
162         2R2O
344         2RF0
650         2RHI
277         2V62
108         2VIF
334         2VRF
257         2VX2
725         2W2I
191         2W3O
215         2W7R
397         2WH0
136         2WUH
57          2X04
1049        2X7K
228         2XHI
74          2XSQ
124         2XZZ
394         2Y7J
5           2YD6
129         2YHW
22          2YLE
56          2YVI
60          2YXM
193         2Z0B
176         2Z0W
428         2Z15
319         2Z5D
701         3A2A
172         3A8Y
440         3AP1
13          3AU4
550         3BC1
120         3BGM
401         3BHD
233         3BOR
275         3BRV
619         3BSU
1249        3BUN
290         3BUX
802         3C3O
1109        3CB2
102         3CBL
246         3CBQ
1209        3CU0
371         3D9J
268         3DAL
393         3DDT
315         3DWD
297         3DYD
87          3DZ8
437         3EBB
271         3EOP
43          3EQ5
656         3EUF
748         3EYI
280         3F6K
1225        3F9X
201         3FBK
528         3FG7
430         3FLV
189         3FQX
243         3FY7
205         3GA3
216         3GLR
746         3GP2
693         3H11
806         3H91
1199        3H95
149         3HFE
417         3HIL
17          3HMH
308         3HNA
805         3HSV
356         3HTU
66          3IEZ
338         3IM4
337         3IQ2
219         3IQU
151         3ISU
72          3IUG
800         3IXE
439         3K6G
523         3KH0
159         3KHF
622         3KNV
361         3KQI
288         3KTR
775         3KUJ
620         3L46
438         3LLH
735         3LLU
1221        3LRR
264         3LTL
369         3LXX
1132        3MEA
424         3MNG
671         3MRE
351         3MRK
554         3MRR
169         3N93
836         3N9U
382         3NH6
421         3NMR
180         3NO8
376         3NWN
433         3O2T
398         3O5Z
1125        3O7A
1057        3OD5
278         3OG6
178         3OSN
105         3P0C
185         3P1N
24          3P6Z
113         3P8C
254         3PFS
84          3PG6
354         3PKN
304         3PLZ
153         3PS4
241         3Q1I
44          3Q3J
390         3Q4A
673         3QIJ
76          3QMD
367         3QX3
143         3RBN
1232        3RBQ
214         3RC4
152         3RC5
114         3RCO
1123        3RD2
310       

###1.5.1 Generation of an intermediate file with the Wild PDB database for all cancers

In [ ]:
df_sort.to_csv("drive/My Drive/MontagemNovaBase/ProcessamentoPDB_Wild/PDB_wild_id_BLCA_LUAD_LUSC.csv",sep='\t',index=False)